src/main/resources/templates/mdBenefitsFlow/householdAdditionalIncomeAmount.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{additional-income.amount.title})}"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+  <section class="slab">
+    <div class="grid">
+      <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <th:block th:replace="~{fragments/icons :: assetsCashAccounts}"></th:block>
+        <th:block
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{additional-income.amount.header})}"/>
+        <th:block
+            th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formDetails})}">
+          <th:block th:ref="formDetails">
+            <div class="form-card__content">
+              <div th:with="income=${inputData.get('additionalIncome[]')},
```

Comment:
I haven't fully flushed this out fully:

How would you feel about adding a method in the utilities file that returns only the list of Additional Income Type that is relevant in this page.

Then you could loop through that list:
```
                  <th:block
                      th:replace="~{fragments/inputs/money :: money(
                          inputName=${addIncType.getInputName().toString()},
                          label=${#messages.msg(addIncType.getLabelSrc())},
                          placeholder=${0.00}
                  )}"/>
                  ```

Reply:
Thanks for catching this - I cleaned the page up.